package.json

```diff
@@ -14,7 +14,7 @@
     "jetpack"
   ],
   "scripts": {
-    "styles": "node-sass data/recommendation/style.scss data/recommendation/style.css --sourcemap=none",
+    "styles": "node-sass data/recommendation/style.scss data/recommendation/style.css --sourcemap=none && node-sass data/recommendation/buttonStyle.scss data/recommendation/buttonStyle.css --sourcemap=none",
```

Comment:
There's probably a better way to do this instead of writing out that full path four times.